{"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"article","variants":[{"paths":["\/documentation\/rxfireauth\/usererror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"role":"collectionGroup","roleHeading":"API Collection","modules":[{"name":"RxFireAuth"}],"title":"LocalizedError Implementations"},"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"identifier":{"url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"sections":[],"references":{"doc://RxFireAuth/documentation/RxFireAuth":{"title":"RxFireAuth","abstract":[],"type":"topic","kind":"symbol","role":"collection","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","url":"\/documentation\/rxfireauth"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/recoverySuggestion":{"type":"topic","title":"recoverySuggestion","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion","url":"\/documentation\/rxfireauth\/usererror\/recoverysuggestion","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?","kind":"text"}],"abstract":[],"kind":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/helpAnchor":{"type":"topic","kind":"symbol","url":"\/documentation\/rxfireauth\/usererror\/helpanchor","title":"helpAnchor","role":"symbol","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor"},"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"type":"topic","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","abstract":[{"text":"Errors thrown by  ","type":"text"},{"code":"UserManagerType","type":"codeVoice"},{"type":"text","text":" implementations."}],"navigatorTitle":[{"text":"UserError","kind":"identifier"}],"title":"UserError","url":"\/documentation\/rxfireauth\/usererror","role":"symbol","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"UserError"}]},"doc://RxFireAuth/documentation/RxFireAuth/UserError/failureReason":{"fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"abstract":[],"type":"topic","title":"failureReason","role":"symbol","url":"\/documentation\/rxfireauth\/usererror\/failurereason","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason"}}}